### DISCUSSION TITLE
Docker compose

### DISCUSSION THREAD

**Discussion by WarmWelcome** [JonathanTreffler/backblaze-personal-wine-container#169] [Ideas]
Any chance of getting a docker compose file for this so that the setup can be more minimal? The opts would be a lot easier to use rather than the copy pasting imo

**Comment by traktuner** [comment]
Hey @WarmWelcome 
thanks for the suggestion - it's on my todo list ;)

**Reply by WarmWelcome** [reply]
Cool, thank you! :)
